app/helpers/scaptimony_policies_helper.rb

```diff
@@ -4,4 +4,26 @@ def profiles_selection
     return @policy.scap_content.scap_content_profiles unless @policy.scap_content.blank?
     return []
   end
+
+  def submit_or_cancel_policy(form, overwrite = nil, args = { })
+    args[:cancel_path] ||= send("#{controller_name}_path")
+    content_tag(:div, :class => "clearfix") do
+      content_tag(:div, :class => "form-actions") do
+        text    = overwrite ? overwrite : _("Submit")
```

Comment:
I mean that the overwrite is not needed, this is a copy paste from the app layout in foreman, which deals with overwrites for host creation (which should not have been global to start with).